scripts/repackage-windows-zip.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-2024  The DOSBox Staging Team
+
+# Repackages the Windows ZIP release package generated by the CI workflow
+# under a parent directory.
+#
+# Prerequisites: zip & unzip commands
+
+set -e
+
+if [ $# -ne 1 ]; then
+	echo "Usage: repackage-windows-zip.sh MSYS2_ZIP_NAME"
+	exit 1
+fi
+
+IN_ZIP=$1
+IN_ZIP_PREFIX=dosbox-staging-windows-msys2-x86_64-
+
+if [[ $IN_ZIP != $IN_ZIP_PREFIX* ]]; then
+	echo "Input filename must start with '$IN_ZIP_PREFIX'"
+	exit 1
+fi
+
+function strip {
+	local STRING=${1#$"$2"}
+	echo "${STRING%$"$2"}"
+}
```

Comment:
I'm probably not the best person to review bash scripts because I don't know what the heck this is doing.  I have an idea based on the context but that syntax is a bit of a yikes.